infra/experimental/chronos/README.md

```diff
@@ -0,0 +1,12 @@
+# Usage
+Under `OSS-Fuzz` root directory:
+```bash
+export PROJECT=libiec61850
+export FUZZ_TARGET=fuzz_mms_decode.c
+export FUZZING_LANGUAGE=c
+infra/experimental/chronos/prepare-recompile "$PROJECT" "$FUZZ_TARGET" "$FUZZING_LANGUAGE"
+python infra/helper.py build_image "$PROJECT"
+docker run -ti --entrypoint="compile" --name "${PROJECT}-origin" "gcr.io/oss-fuzz/${PROJECT}"
```

Comment:
when `compile` is run here, `SANITIZER` wont be set and consequently we won't set the any sanitizer flags in the following code (since sanitizer is `''`) (https://github.com/google/oss-fuzz/blob/432e69a594f7d9f019b12047271b5235f3b252b2/infra/base-images/base-builder/compile#L69-L72) as we want the relevant variables here https://github.com/google/oss-fuzz/blob/432e69a594f7d9f019b12047271b5235f3b252b2/infra/base-images/base-builder/Dockerfile#L60-L76

For OFC we need to have two different builds:
SANITIZER == 'coverage'
SANITIZER == 'address'
but I assume in general it's intended to use this with sanitizers, which might make it nice to show how to do this in the README. I think just adding `-e SANITIZER="address"` to the `docker run` command would do the job

Reply:
@DonggeLiu re https://github.com/google/oss-fuzz/pull/11937#issuecomment-2274429272 -- I think my comment only applies to the README for now

Reply:
OK Done